### DISCUSSION TITLE
any visualizer suitable for interactive visualizing sparql queries?

### DISCUSSION THREAD

**Discussion by shizhaojingszj** [oxigraph/oxigraph#396] [Q&A]
I need an interactive visualizer (best web based) for oxigraph data. Any suggestions?

**Comment by Tpt** [comment]
That's a great question. Most of the visualizer based on SPARQL queries should be able to work with Oxigraph.

For example:
- https://github.com/MadsHolten/sparql-visualizer
- https://triply.cc/docs/yasgui/ (the UI already provided by Oxigraph server)
